src/nvim/window.c

```diff
@@ -6394,6 +6394,10 @@ void restore_win_noblock(win_T *save_curwin, tabpage_T *save_curtab,
   if (win_valid(save_curwin)) {
     curwin = save_curwin;
     curbuf = curwin->w_buffer;
+  } else if (curwin->w_floating) {
```

Comment:
I am not sure this should be done. floating windows can be entered as ordinary windows in neovim, unlike vim popups.

Reply:
I'll consider it N/A then. Up to you to port if needed.